lib/vdc/BOVCollection.cpp

```diff
@@ -132,8 +147,9 @@ int BOVCollection::_parseHeader(std::ifstream &header)
         rc = _findToken(DATA_FILE_TOKEN, line, dataFile);
         if (rc == (int)parseCodes::ERROR)
             return _failureToReadError(DATA_FILE_TOKEN);
-        else if (rc == (int)parseCodes::FOUND)
+        else if (rc == (int)parseCodes::FOUND) {
             _dataFile = dataFile;
+        }
```

Comment:
In the case when you do find a token from the current line, do you expect to find another token from the same line? If not, would it make sense to skip the rest of the token detection code in the current iteration?

Reply:
Nope, fixed.